packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -110,7 +111,7 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = (
                 background: neutralFillActive,
             },
             ...applyFocusVisible({
-                borderColor: designSystem.foregroundColor, // TODO Need to create focus utilities
+                borderColor: neutralFocus(designSystem),

```

Comment:
you don't have to call the function here if you don't want - just like the others, you can simply assign the function: `borderColor: neutralFocus`